tests/integration/security/authorization_test.go

```diff
@@ -1066,3 +1069,56 @@ func TestAuthorization_Path(t *testing.T) {
 			rbacUtil.RunRBACTest(t, cases)
 		})
 }
+
+//Test that LOG action does not impact authorization
```

Comment:
`// Test that LOG action does not impact authorization of allowing or denying a request.`